images/spire/config/main.tf

```diff
@@ -43,6 +44,6 @@ output "config" {
     entrypoint = {
       command = var.command
     }
-    cmd = var.cmd
+    cmd = var.entrypoint-cmd
```

Comment:
This should be the other way around. It's `var.command` that goes to the entrypoint bit, so that variable should be called `entrypoint-*` instead of this one.

Reply:
Oh okay thanks for this! I got confused, just fixing it